pucauto.py

```diff
@@ -8,11 +8,12 @@
 from datetime import datetime
 from bs4 import BeautifulSoup
 
+from lib import logger
 
 with open("config.json") as config:
     CONFIG = json.load(config)
 
-
+LOGGER = logger.get_default_logger(__name__)
```

Comment:
This gets a "default logger" (i.e., one set up with a bunch of pre configured values in `lib/logger.py`.

Passing in `__name__` to it will be much more handy later. You can pull in a new logger in...say, `lib/trades.py`, and all logging messages created in that module will be logged with the name `lib.trades` (or something similar).

I feel that personally, `LOGGER` as a constant isn't 100% accurate. This is still an instance of a logging interface, and to think about it as a constant doesn't make sense. `logger` is a much more suitable (and common) name for this.